File: python/jzoffer-8.py
# -*- coding:utf-8 -*-
class Solution:
    def jumpFloor(self, number):
        # write code here
        if number == 0:
            return 0
        x,a,b = 0,0,1
        while x < number:
            a,b = b,a+b
            x+=1
        return b

File: java/jzoffer-8.java
public class Solution {
    public int JumpFloor(int target) {
        if(target == 0){
            return 0;
        }
        int x = 0,a = 0,b = 1;
        int i = 0;
        int j = 0;
        while(x < target){
            i = b;
            j = a + b;
            a = i;
            b = j;
            x += 1;
        }
        return b;
    }
}